pyls/workspace.py

```diff
@@ -93,9 +94,12 @@ def source_roots(self, document_path):
 
     def _create_document(self, doc_uri, source=None, version=None):
         path = uris.to_fs_path(doc_uri)
+        extra_sys_path = self.source_roots(path)
+        settings = self._config.settings(doc_uri)
+        extra_sys_path.extend(settings.get('extraSysPath', []))
```

Comment:
Extend? Or prepend? I wonder if an explicit config override should be put at the front of the sys path